migrations/20190808050641-add-device-rename-history.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const util = require("../models/util/util");
+
+const tableName = "DeviceHistories";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(tableName, {
+      oldName: { type: Sequelize.STRING, allowNull: false },
+      newName: { type: Sequelize.STRING, allowNull: false },
+      oldGroupID: { type: Sequelize.INTEGER, allowNull: false },
+      newGroupID: { type: Sequelize.INTEGER, allowNull: false },
+      createdAt: { type: Sequelize.DATE, allowNull: false },
+      updatedAt: { type: Sequelize.DATE, allowNull: false },
+    }).then(() => {
+      return util.addSerial(queryInterface, tableName);
+    }).then(() => {
+      return util.migrationAddBelongsTo(queryInterface, tableName, "Devices");
+    });
+  },
+
+  down: (queryInterface) => {
+    return queryInterface.sequelize.query('DROP TABLE "'+tableName+'" CASCADE');
```

Comment:
I don't think the CASCADE here adds any value because no table refers to `DeviceHistories` (only the other way around).